blockstack-sdk/src/main/java/org/blockstack/android/sdk/BlockstackSession.kt

```diff
@@ -65,10 +68,18 @@ class BlockstackSession(context: Context? = null, private val config: Blockstack
     private var getAppBucketUrlCallback: ((Result<String>) -> Unit)? = null
     private var getUserAppFileUrlCallback: ((Result<String>) -> Unit)? = null
 
-
-    private val v8blockstackAndroid: V8Object
+    internal val v8blockstackAndroid: V8Object
     private val v8userSessionAndroid: V8Object
+    private val v8networkAndroid: V8Object
     private val v8userSession: V8Object
```

Comment:
This should be camelCased, and is "Android" necessary in the name?
Also, would it make sense to have the object type at the end of the name?

i.e.
blockstackV8
networkV8

What is the difference between v8UserSession and v8UserSessionAndroid?

Reply:
The names reflect the location in the javascript world. The prefix indicate that they are v8 object and have a correspondence in there. Then name that follows is the same in javascript. So
v8networkAndroid in Android is the representation of networkAndroid object in javascript.

All the *Android javascript objects are defined in blockstack_android.js. 

userSession is the blockstack object that can be called directly if the method signatures are transferrable between the two worlds, userSessionAndroid is the bridge object that translates `Promise`s and the like into String, etc. These *Android object make also calls to the java world.

Reply:
Thanks for explaining.